packages/internal/src/action-types.js

```diff
@@ -15,3 +15,4 @@ export const VBANK_BALANCE_UPDATE = 'VBANK_BALANCE_UPDATE';
 export const WALLET_ACTION = 'WALLET_ACTION';
 export const WALLET_SPEND_ACTION = 'WALLET_SPEND_ACTION';
 export const INSTALL_BUNDLE = 'INSTALL_BUNDLE';
+export const VTRANSFER_IBC_EVENT = 'VTRANSFER_IBC_EVENT';
```

Comment:
I'd like to see a comment above declaration of `IBC_EVENT` that following exports describe internal events for which handling code is expected in packages/cosmic-swingset/src/launch-chain.js.

Reply:
I'm not sure why this action needs a comment any more than any other of the actions above? I'd be ok restructuring to highlight which actions come through the queue and which come through the pipe.